### PR TITLE
fixed responsive issue on minicart and search suggestion

### PR DESCRIPTION
**Pull request by asad-rafter**

### CODE REVIEW

File: cartridges/int_cloudinary_sfra/cartridge/client/default/js/components/miniCart.js [hunk 1]
```diff
@@ -2,12 +2,18 @@
...
-            if (typeof window.makeCloudinaryImagesResponsive !== 'undefined') {
-                window.makeCloudinaryImagesResponsive();
-            }
-        }
-    });
+	const targetNode = document.querySelector('.minicart');
+	const config = { childList: true, subtree: true };
+
+	const callback = function () {
+		if (targetNode.innerHTML.length > 0) {
```
Comment (yuval-cloudinary) [inline cartridges/int_cloudinary_sfra/cartridge/client/default/js/components/miniCart.js:9]: please merge consecutive if statements

File: cartridges/int_cloudinary_sfra/cartridge/client/default/js/components/miniCart.js [hunk 2]
```diff
@@ -2,12 +2,18 @@
 
 module.exports = function () {
     // Custom Start: Make cart product images responsive //
```
Comment (yuval-cloudinary) [inline cartridges/int_cloudinary_sfra/cartridge/client/default/js/components/miniCart.js:4]: is it possible that the minicart would be called twice, and we will create duplicate observers?
Maybe it makes sense to use a global DOM variable to skip a second call to the minicart

File: cartridges/int_cloudinary_sfra/cartridge/client/default/js/components/miniCart.js [hunk 3]
```diff
@@ -2,12 +2,18 @@
...
+
+	const callback = function () {
+		if (targetNode.innerHTML.length > 0) {
+			if (typeof window.makeCloudinaryImagesResponsive !== 'undefined') {
+				window.makeCloudinaryImagesResponsive();
+			}
+		}
+	};
+
+	const observer = new MutationObserver(callback);
```
Comment (yuval-cloudinary) [inline cartridges/int_cloudinary_sfra/cartridge/client/default/js/components/miniCart.js:16]: MutationObserver - I like it!

**Review by yuval-cloudinary — CHANGES_REQUESTED** [review]

File: cartridges/int_cloudinary_sfra/cartridge/client/default/js/components/miniCart.js (hunk 2 above)
Comment (asad-rafter) [reply cartridges/int_cloudinary_sfra/cartridge/client/default/js/components/miniCart.js:4]: It only call once.

**Review by yuval-cloudinary — APPROVED** [review]

**Pull request merged**